src/v1/sources/cordial/mapping.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "sourceKeys": "event._id",
+    "destKeys": "properties.event_id"
+  },
+  {
+    "sourceKeys": "contact._id",
+    "destKeys": ["context.traits.userId", "userId"]
+  },
+  {
+    "sourceKeys": ["event.ts", "event.ats"],
+    "destKeys": ["timestamp", "sentAt", "originalTimestamp"]
+  },
+  {
+    "sourceKeys": "event.d",
+    "destKeys": "context.device"
+  },
+  {
+    "sourceKeys": "contact.channels.email.address",
+    "destKeys": ["context.traits.email", "properties.email"]
```

Comment:
Why are we setting again in `properties` when are already setting in `context.traits` ?

Reply:
done . Removed "properties.email"